src/types/track.ts

```diff
@@ -18,6 +18,12 @@ export type NFTTrackJoin = {
   processedTrackId: string;
 }
 
+export type Artworks = {
```

Comment:
i can make a new file for this type if required

Reply:
np whichever

trackId and error seem missing?

Reply:
also should be singular, just Artwork?

Reply:
probably should be singular